cap/config.py

```diff
@@ -91,9 +91,23 @@ def _(x):
 #: Email address for support.
 SUPPORT_EMAIL = "analysis-preservation-support@cern.ch"
 #: Disable email sending by default.
-MAIL_DEBUG = False
+CAP_SEND_MAIL = False
+
+# Flask-mail variables
 MAIL_SUPPRESS_SEND = True
 
+#: use appropriate server/port pair here
```

Comment:
```
#: use appropriate server/port pair here
 MAIL_SERVER = 'localhost'
 MAIL_PORT = 25

  #: change depending on the server (CERN uses TLS)
 MAIL_USE_TLS = False
 MAIL_USE_SMTP = False

  #: username needs to be a full e-mail address
 MAIL_USERNAME = None
 MAIL_PASSWORD = None
```

All these are defaults from `Flask-Mail` no need to specify here..To be removed

Please add comment/docs instead for how to setup with CERN email service